nservicebus/wormhole/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Wormhole Gateway
+summary: How to send messages between geographically separate sites using Worm Hole Gateway 
+component: Wormhole
+reviewed: 2017-05-20
+---
+
+`NServiceBus.Wormhole` is an alternative to the [Gateway](/nservicebus/gateway/) component of NServiceBus. Following table compares the most important aspects of both technologies.
+
+|                         | Gateway                                 | Wormhole                                                                                |
+|-------------------------|-----------------------------------------|-----------------------------------------------------------------------------------------|
+| Encryption              | transport-level                         | no                                                                                      |
+| Transports              | <ul><li>HTTP</li><li>custom</li></ul>   | <ul><li>all NServiceBus transports</li><li>HTTP</li></ul>                               |
```

Comment:
This row could be (and avoid bullets):

Transports | HTTP / Custom | All NServiceBus Transports / HTTP